sdk/identity/azure-identity/azure/identity/_credentials/linux_vscode_adapter.py

```diff
@@ -0,0 +1,65 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+import os
+import json
+import ctypes as ct
+from .._constants import VSCODE_CREDENTIALS_SECTION
+
+
+def _c_str(string):
+    return ct.c_char_p(string.encode('utf-8'))
+
+
+try:
+    _libsecret = ct.cdll.LoadLibrary('libsecret-1.so.0')
+    _libsecret.secret_schema_new.argtypes = \
+        [ct.c_char_p, ct.c_uint, ct.c_char_p, ct.c_uint, ct.c_char_p, ct.c_uint, ct.c_void_p]
+    _libsecret.secret_password_lookup_sync.argtypes = \
```

Comment:
might want to set the restype
https://docs.python.org/3/library/ctypes.html#return-types

Reply:
Added.